cmake/projects/libigl/hunter.cmake

```diff
@@ -15,18 +15,18 @@ hunter_add_version(
     PACKAGE_NAME
     libigl
     VERSION
-    2.0.0-e33f534-p0
+    2.0.0-e33f534-p1
```

Comment:
Please add a new entry, no need to remove old:
- https://docs.hunter.sh/en/latest/creating-new/update.html#update-package

Reply:
Fixed, although the previous version is not working.